cloudbuild.yaml

```diff
@@ -0,0 +1,7 @@
+steps:
+  - name: 'gcr.io/cloud-builders/mvn'
+    id: DEPLOY
+    args: ['-Ddocker.image.prefix=gcr.io/ceres-dev-222017', 'package', 'jib:dockerBuild']
+
+images:
+  - 'gcr.io/ceres-dev-222017/tenant-routing-service'
```

Comment:
This build is locked into the "dev" cluster. This should use `$PROJECT_ID` like the other builds, such as

https://github.com/racker/salus-telemetry-ambassador/blob/master/cloudbuild.yaml#L13

Reply:
Yes, but we don't want to build images for each env. So the images will be stored in the dev project and pulled from here for all envs.